src/sbvr-api/sbvr-utils.coffee

```diff
@@ -721,6 +780,8 @@ exports.handleODataRequest = handleODataRequest = (req, res, next) ->
 					else
 						{ req, request } = requestEnv
 						runRequest(req, res, tx, request)
+				.tapCatch ->
```

Comment:
This needs to be unindented a bunch, because right now it's won't roll back PREPARSE/POSTPARSE hooks if the failure happens during translation/parsing/etc